DataCapturing/Source/Model/Migrations/CoreDataManager.swift

```diff
@@ -73,24 +73,20 @@ public class CoreDataManager {
        - migrator: An object to migrate between different Cyface model versions.
      - Throws: `CoreDataError.invalidModelUrl`, `CoreDataError.modelNotAvailable`
      */
-    public init(storeType: String = NSSQLiteStoreType, migrator: CoreDataMigratorProtocol = CoreDataMigrator()) throws {
+    public convenience init(storeType: String = NSSQLiteStoreType, migrator: CoreDataMigratorProtocol = CoreDataMigrator()) throws {
+        let momdName = "CyfaceModel"
+        let mom = try CoreDataManager.loadModel()
+        self.init(storeType: storeType, migrator: migrator, modelName: momdName, model: mom)
+    }
+
+    init(storeType: String = NSInMemoryStoreType, migrator: CoreDataMigratorProtocol = CoreDataMigrator(), modelName: String, model: NSManagedObjectModel) {
```

Comment:
Appearantly recreating the data model multiple times is not a good idea. This is no problem in the actual iOS Application as that creates the data model only a single time. But our tests used to create it for every single test run. These (and the next two) changes enable the creation of a data model outside the CoreDataManager and providing the same data model over and over again, which should reduce warnings and performance overhead during test runs.